### DISCUSSION TITLE
Data verloren van 2 dagen🙋

### DISCUSSION THREAD

**Discussion by Roel0412** [dsmrreader/dsmr-reader#1629] [Questions & answers / How-to's] [answered]
### Description

In de nacht van vrijdag op zaterdag is er een storing ontstaan op mijn raspberry pi. Sindsdien heeft DSMR reader niets opgeslagen aan data. Na een herstart van de Raspberry Pi vandaag werkt alles weer, maar ben nu de data kwijt van zaterdag, zondag en de halve dag vandaag.

Kan dit nog ergens uitgelezen worden, zodat de data beschikbaar komt?

### DSMR-reader version

5.1

**Comment by dennissiemensma** [comment, marked as answer]
Bedankt voor je vraag. Helaas is de data in dat geval weg. Als DSMR-reader niets heeft kunnen schrijven, staat het ook nergens.
Hier is ook vrij weinig aan te doen. Je kunt in theorie bijvoorbeeld monitoring toevoegen, maar dan moet je monitoring ook blijven werken. Etc, etc.

Ik denk overigens dat je van geluk mag spreken dat je er zo relatief snel achter bent gekomen ipv na enkele maanden.

**Reply by Roel0412** [reply]
Dat is jammer. Normaal kijk ik wel een paar keer per week naar de gegevens, dus zo'n groot probleem is het niet. 

Bedankt voor je antwoord in ieder geval!

Nog wel een andere vraag, is het mogelijk om handmatig nog data toe te voegen? Als ik bij mijn energieleverancier kijk zie ik wel de gegevens staan en kan ik van daaruit de missende gegevens invullen (als dat kan).

Edit: Het is gelukt om de data van het verbruik en teruglevering handmatig in te vullen.
